template-only-bin/install-template.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -euo pipefail
+
+echo "Fetch latest version of template-application-nextjs"
+git clone --single-branch --branch main --depth 1 git@github.com:navapbc/template-application-nextjs.git
+
+echo "Copy files from template-application-nextjs"
+cd template-application-nextjs
+cp -r \
+  .github \
+  docs \
+  app \
+  docker-compose.yml \
+  renovate.json \
```

Comment:
copied renovate.json over here as part of install script. curious, is it meant to be in top level of repo? could one renovate.json file be used for repos with multiple apps, or is it app specific?

Reply:
I haven't tried it but *I think* it's possible to have multiple config files per repo https://docs.renovatebot.com/configuration-options/

How does dependabot work in this respect? 🤔